components/eam/src/physics/cam/zm_conv_intr.F90

```diff
@@ -173,6 +174,74 @@ subroutine zm_conv_init(pref_edge)
     call addfld ('ZMICUD',     (/ 'lev' /), 'A', 'm/s', 'ZM in-cloud U downdrafts')
     call addfld ('ZMICVU',     (/ 'lev' /), 'A', 'm/s', 'ZM in-cloud V updrafts')
     call addfld ('ZMICVD',     (/ 'lev' /), 'A', 'm/s', 'ZM in-cloud V downdrafts')
+
+    ! Add fields for ZM stand-alone tests
```

Comment:
I'm usually not a fan of leaving commented out code, this will likely go out of date, and may never end up being used if we abandon the fortran version of the code.

Reply:
Fair point.  I struggled with keeping this or not.  Happy to delete.